Fix duplicate prefix on CF data flow server configuration properties

I started seeing this compilation error on `master` on my fork branch:

```
[ERROR] /Users/igopinathan/dev/git/ilayaperumalg/spring-cloud-dataflow/spring-cloud-dataflow-platform-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/config/cloudfoundry/CloudFoundryDataFlowServerConfiguration.java:[48,58] Duplicate `@ConfigurationProperties` definition for prefix 'spring.cloud.dataflow.server.cloudfoundry'
```
though this built ok from my remote branch
looks like the issue is real as we don't need explicit configuration properties defined in [here](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-platform-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/config/cloudfoundry/CloudFoundryDataFlowServerConfiguration.java#L47)

--- a/spring-cloud-dataflow-platform-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/config/cloudfoundry/CloudFoundryDataFlowServerConfiguration.java
+++ b/spring-cloud-dataflow-platform-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/server/config/cloudfoundry/CloudFoundryDataFlowServerConfiguration.java
@@ -44,7 +44,6 @@ public class CloudFoundryDataFlowServerConfiguration {
 	}
 
 	@Bean
-	@ConfigurationProperties(prefix = CloudFoundryServerConfigurationProperties.PREFIX)
 	public CloudFoundryServerConfigurationProperties cloudFoundryServerConfigurationProperties() {
 		return new CloudFoundryServerConfigurationProperties();
 	}

